README.md

```diff
@@ -87,6 +87,42 @@ bin/br restore full --pd pd0:2379 --storage "s3://mybucket/full" \
     --s3.endpoint="$S3_ENDPOINT"
 ```
 
+## Tiup Quick Start
+
+```sh
+# Using tiup to start a TiDB cluster.
+tiup playgroup
+
+# Using tiup bench to generater test data.
+tiup bench tpcc --warehouses 1 prepare
+
+# How many row do we get? 300315 rows.
+mysql --host 127.0.0.1 --port 4000 -E -e "SELECT COUNT(*) FROM test.customer" -u root -p
+
+# Build br.
+make build
+
+# Backup TPC-C test data.
+bin/br backup table --db test \
+    --table order_line \
+	-s local:///mnt/d/backup_test/ \
```

Comment:
How about backup to `/tmp/backup_test`? This path seems only exists at WSL environment.